src/singletons/Settings.hpp

```diff
@@ -189,17 +187,9 @@ class Settings
 
     //    BoolSetting collapseLongMessages =
     //    {"/appearance/messages/collapseLongMessages", false};
-    QStringSetting chatFontFamily{
-        "/appearance/currentFontFamily",
-        DEFAULT_FONT_FAMILY,
-    };
-    IntSetting chatFontSize{
-        "/appearance/currentFontSize",
-        DEFAULT_FONT_SIZE,
-    };
-    IntSetting chatFontWeight = {
-        "/appearance/currentFontWeight",
-        QFont::Normal,
+    QStringSetting chatFont{
+        "/appearance/chatFont",
+        DEFAULT_FONT,
```

Comment:
I'd prefer if you can keep the same 3 settings.
Otherwise we'll have to do our world first setting migration (which I have some uncommited code for already).